MoreLinq/tt.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+[[ -e tt.sh ]] || { echo >&2 "Please cd into the script location before running it."; exit 1; }
+set -e
+find . -name "*.tt" | xargs -t -n 1 dotnet tt
```

Comment:
The canonical option is `-L 1` (`-n` has some special handling for the case the argument is larger than 1 but there are not enough arguments, and is not relevant for this particular case). I would use `-L` if only because it is more likely to be recognized already (I had to look up `-n`).

Also, on these pipelines, it is customary to pass `-print0` to `find` and `-0` to `xargs`, to prevent files with whitespace or other oddities to mess the `xargs` line parsing.

Reply:
> Also, on these pipelines, it is customary to pass `-print0` to `find` and `-0` to `xargs`, to prevent files with whitespace or other oddities to mess the `xargs` line parsing.

Am aware but wanted to keep it consistent with the Windows script for now, as I'm not sure I designed that to work with whitespaces in names. I can test and revert.